задача Турнир в Королевской гавани на проверку

samekeyev opened the pull request.

samekeyev committed:
задача Турнир в Королевской гавани на проверку

samekeyev committed:
исправил названия переменных рыцарей на более понятные

SnurnicynKonstantin commented on src/main/java/bjs2_89586/King.java:
```diff
@@ -0,0 +1,31 @@
+    public void startTournament() {
+        ExecutorService executorService = Executors.newFixedThreadPool(THREAD_POOL_SIZE);
+
+        knightTrials.forEach((knight, trials) -> {
```
Можно на стрим переделать:
```
knightTrials.values().stream()
    .flatMap(List::stream)
    .map(executorService::submit)
    .collect(Collectors.toList());
```

samekeyev replied:
вместо
```
.map(executorService::submit)
.collect(Collectors.toList());
```
использовал
`.forEach(executorService::submit);`
т.к. возвращаемое значение не нужно

samekeyev committed:
Исправил замечания, переделал на стрим
вместо
.map(executorService::submit)
.collect(Collectors.toList());
использовал
.forEach(executorService::submit);
т.к. возвращаемое значение не нужно
